docs/how-to-work-on-the-docs-theme.md

```diff
@@ -44,19 +44,7 @@ Typically you would not need to change any configuration or build the site local
 
 ### Serving the documentation site locally
 
-Clone freeCodeCamp:
-
-```console
-git clone https://github.com/freeCodeCamp/freeCodeCamp.git
-```
-
-Install `docsify`:
-
-```console
-npx docsify serve docs
-```
-
-Alternatively, if you have installed freeCodeCamp locally (see the local setup guide), we bundled the CLI with the development tools so you can run any of the below commands as needed from the root of the repo:
+Install freeCodeCamp locally ([see the local setup guide](how-to-setup-freecodecamp-locally)), we bundled the CLI with the development tools so you can run any of the below commands as needed from the root of the repo:
```

Comment:
We don't need to mention `docsify` anymore? Couldn't find its section on how to setup freecodecamp https://github.com/freeCodeCamp/freeCodeCamp/pull/48477/files#diff-a83f7d3a1265cf77443308d92a9055f0accd2606762513edd86c6a0f3c28c90a

Reply:
Read the guide thoroughly (without skipping) again to understand the change: https://contribute.freecodecamp.org/#/how-to-work-on-the-docs-theme?id=serving-the-documentation-site-locally

Reply:
I have read it and I know about `npm run docs:serve` open port 3400, I am talking about the point of running the docs without install freecodecamp. 

My laptop isn't powerful to test if `npx docsify serve docs` will run the docs without install freecodecamp, that why I asked

Reply:
Give me a minute I can test in Github codespace

Reply:
The `npm run docs:serve` assumes the repository is "setup" - that is literally what the instruction says in the opening line.

Two ways you are confusing yourself here:

1. `npx ...` - You don't need `npx` because `docsify` CLI has always been included in the dev-deps.
2. Debugg the error – what do you think `...docsify: not found` means? What does it say about your setup?

Feel free to continue discussing and debugging in the chat.

Reply:
https://user-images.githubusercontent.com/1884376/201346679-cc6a3f37-c413-4e30-985a-fa74bcd098aa.mp4

Reply:
> The `npm run docs:serve` assumes the repository is "setup" - that is literally what the instruction says in the opening line.
> 
> Two ways you are confusing yourself here:
> 
> 1. `npx ...` - You don't need `npx` because `docsify` CLI has always been included in the dev-deps.
> 2. Debug the error – what do you think `...docsify: not found` means? What does it say about your setup?
> 
> Feel free to continue discussing and debugging in the chat.

No need, and thank you for the gif. I thought the intent of that section was to run the docs without running `npm install`, that's why I changed it to `npx docsify serve docs` following these docs https://docs.npmjs.com/cli/v7/commands/npx

I can't think of a way to run the docs locally without installing the dependency, so I resolved the conversation

Reply:
How I though the intent was to run without installing the local dependency, I have no idea what I have read wrong :shrug:

Reply:
I'll bite and explain. I did not mean to rush you.

> intent was to run without installing the local dependency

You are correct - that was the idea when the guide was first written (I wrote it). But if you look at recent workflows, you either use GitPod or Locally Clone (a very large download).

With GitPod, the deps are installed automatically. So there is no point in `npx` or global install. You would use the run-scripts.

With a local clone, if you have bothered to get the repo on the computer, how unwilling will the contributor be to run these:

```
npm ci
npm run docs:serve
```

I do not think so.

Finally, if you are the laziest of the contributors, you will not care and use GitHub anyways. 

What I did with the change was hold the hand a bit more. 

See, people will still not read and do what they want and then land in the chat with 'how do I X?' questions.

Lol.

Reply:
> You are correct - that was the idea when the guide was first written (I wrote it). But if you look at recent workflows, you either use GitPod or Locally Clone (a very large download).
>
> With GitPod, the deps are installed automatically. So there is no point in npx or global install. You would use the run- scripts.
>
> With a local clone, if you have bothered to get the repo on the computer, how unwilling will the contributor be to run these:

True, I see your point 👍 😆